api/python/t4/api.py

```diff
@@ -141,7 +141,7 @@ def list_packages(registry=None):
         A list of strings containing the names of the packages        
     """
     if not registry:
-        registry = ''
+        registry = 'whatever'
```

Comment:
Obvious question: what _should_ this be? Probably not the current dir. `get_local_package_registry()`? We need to refactor some of that code.

Reply:
why not the default local one?